inst/CITATION

```diff
@@ -2,19 +2,20 @@ citHeader("To cite LPWC in publications use:")
 
 citEntry(entry = "Article",
   title        = "Lag Penalized Weighted Correlation for Time Series Clustering",
-  author       = personList(as.person("Thevaa Chandereng"),
-                   as.person("Anthony Gitter")),
-  journal      = "bioRxiv",
-  year         = "2018",
-  pages        = "292615",
-  url          = "https://doi.org/10.1101/292615",
-  publisher    = "Cold Spring Harbor Laboratory",
-  doi          = "10.1101/292615",
+  author       = "Thevaa Chandereng and Anthony Gitter",
```

Comment:
Was the `personList` and `as.person` formatting important before?  What does that do?

Reply:
I planned to check the website on this once we pull it to master. The only issue would be with the website we have. Otherwise, it should work.

Reply:
Okay, we can wait for the website to build after merging this to confirm everything looks good before submitting to CRAN.